common/web/types/src/keyman-touch-layout/keyman-touch-layout-file.ts

```diff
@@ -0,0 +1,87 @@
+//
+// .keyman-touch-layout JSON format
+//
+// Follows /common/schemas/keyman-touch-layout/keyman-touch-layout.spec.json for
+// reading and
+// /common/schemas/keyman-touch-layout/keyman-touch-layout.clean.spec.json for
+// writing
+//
+
+export interface TouchLayoutFile {
+  tablet?: TouchLayoutPlatform;
+  phone?: TouchLayoutPlatform;
+  desktop?: TouchLayoutPlatform;
+};
+
+export type TouchLayoutFont = string;
+export type TouchLayoutFontSize = string;
+export type TouchLayoutDefaultHint = "none"|"dot"|"longpress"|"multitap"|"flick"|"flick-n"|"flick-ne"|"flick-e"|"flick-se"|"flick-s"|"flick-sw"|"flick-w"|"flick-nw";
```

Comment:
This reminds me that the 'directions' schema should have a regex to validate this.

Would it be helpful in general to have supplemental metadata with a direction list?

Reply:
Started a DTD PR https://github.com/unicode-org/cldr/pull/2437 - now back to this review

Reply:
> Would it be helpful in general to have supplemental metadata with a direction list?

I think that given we are only supporting unidirectional flicks at this point, it doesn't help all that much.